stacks-signer/src/client/stackerdb.rs

```diff
@@ -57,16 +59,15 @@ impl StackerDB {
     ) -> Result<StackerDBChunkAckData, ClientError> {
         let message_bytes = bincode::serialize(&message).unwrap();
         let slot_id = message.slot_id(id);
-
         loop {
             let slot_version = *self.slot_versions.entry(slot_id).or_insert(0) + 1;
             let mut chunk = StackerDBChunkData::new(slot_id, slot_version, message_bytes.clone());
             chunk.sign(&self.stacks_private_key)?;
             debug!("Sending a chunk to stackerdb!\n{:?}", &chunk);
             let send_request = || {
-                self.signers_stackerdb_session
-                    .put_chunk(&chunk)
-                    .map_err(backoff::Error::transient)
+                let result = self.signers_stackerdb_session.put_chunk(&chunk);
+                println!("result: {:?}", result);
```

Comment:
Maybe use `slog_debug` & `debug!` here?

Reply:
oh this is a WIP merge. Broken. Just for testing. Will def not be keeping this in :)